src/Arcus.BackgroundJobs.DataBricks/DatabricksInfoProvider.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Azure.Databricks.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.BackgroundJobs.Databricks
```

Comment:
Nit - Can you rename the folder given it is called `DataBricks` instead of `Databricks?

Reply:
Yes, sorry about that.